scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -250,8 +250,8 @@
           <a href="http://database.liulab.science/dbNSFP" target="_blank" data-bs-toggle="tooltip"
              title="REVEL scores were ranked among all REVEL scores in dbNSFP. The REVEL rank score is the ratio of the rank of the score over the total number of REVEL scores in dbNSFP">REVEL rank score</a>
           <span class="float-end">
-          {% if variant.revel_score  %}
-             {{ variant.revel_score }}
+          {% if variant.revel  %}
```

Comment:
Hm, but this does not make sense? We used the revel rank score exclusively before, named `revel_score`. The raw score was added after Hans remarked some guidelines used it.

Reply:
Ah, not `revel_score` everywhere.. It swaps names.. 😆

Reply:
I'm confused. Where the variant is parsed the code is like this:

<img width="637" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/062f4349-5d6d-4c86-8937-092ca0208e36">

<img width="1332" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/b4b7a5e3-08ba-4592-a501-6741b77219fc">

They seem swapped in tto me in the template? 🤔

Reply:
Ah, but no. The transcript has `REVEL_RANKSCORE` stored as `revel` (parse/variant/transcript.py:43). Then the variant has it as `revel_score` parse/variant/transcript.py:239. 

Next time, more informative variable names, and no swaps with an old name. 😬

Reply:
Ah right I missed when we swapped them in the parse/variant/transcriipt 🤣 

<img width="504" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/5f63cf36-c7ce-4347-aba5-ff538c299096">

But perhaps to make it easier by swapping the keys where they are first created (parse/variant/transcript.py:43). Let's see

Reply:
Yes, it is not exactly kind to the reader. 😆 Just don't go changing the keys for anything that gets stored on db; we definitely don't also want to have legacy cases around with another nomenclature.

Reply:
Or, well, if so, make it a completely new name so we can ask for both.